src/software/backend/simulation/BUILD

```diff
@@ -43,3 +45,28 @@ cc_test(
         "@gtest//:gtest_main",
     ],
 )
+
+cc_library(
+    name = "field",
```

Comment:
can we match the library name to the files just to be consistent with how we've done it elsewhere in the repo?

Reply:
agree. Changed this for Physics Ball as well